src/DataWrangling/restoring.jl

```diff
@@ -19,50 +20,48 @@ import Oceananigans.OutputReaders: new_backend, update_field_time_series!
 @inline instantiate(T::DataType) = T()
 @inline instantiate(T) = T
 
-struct ECCONetCDFBackend{N, C, I, M} <: AbstractInMemoryBackend{Int}
+struct NetCDFBackend{N, C, I, M} <: AbstractInMemoryBackend{Int}
     start :: Int
     length :: Int
     inpainting :: I
     metadata :: M
 
-    function ECCONetCDFBackend{N, C}(start::Int, length::Int, inpainting, metadata) where {N, C}
+    function NetCDFBackend{N, C}(start::Int, length::Int, inpainting, metadata) where {N, C}
         M = typeof(metadata)
         I = typeof(inpainting)
         return new{N, C, I, M}(start, length, inpainting, metadata)
     end
 end
 
-Adapt.adapt_structure(to, b::ECCONetCDFBackend{N, C}) where {N, C} = ECCONetCDFBackend{N, C}(b.start, b.length, nothing, nothing)
+Adapt.adapt_structure(to, b::NetCDFBackend{N, C}) where {N, C} = NetCDFBackend{N, C}(b.start, b.length, nothing, nothing)
 
 """
-    ECCONetCDFBackend(length, metadata;
-                      on_native_grid = false,
-                      cache_inpainted_data = false,
-                      inpainting = NearestNeighborInpainting(Inf))
+    NetCDFBackend(length, metadata;
+                  on_native_grid = false,
+                  cache_inpainted_data = false,
+                  inpainting = NearestNeighborInpainting(Inf))
 
-Represent an ECCO FieldTimeSeries backed by ECCO native netCDF files.
+Represent a FieldTimeSeries backed by native netCDF files.
 Each time instance is stored in an individual file.
 """
-function ECCONetCDFBackend(length, metadata;
+function NetCDFBackend(length, metadata;
```

Comment:
code alignment is clunky here

(this is a generic comment, applies to many places...)